recipe/meta.yaml

```diff
@@ -36,6 +36,7 @@ source:
     - 0306-Fix-absl-linker-error.patch
     - 0307-Fix-for-eigen-TensorReduction.patch           #[x86_64]
     - 0308-Upstream-patch-to-disable-absl-cord.patch     #[cudatoolkit == "10.2"]
+    - 0309-disable-ppc64le-custom-gemm_pack_rhs-to-use-the-convolution-specialization
```

Comment:
Can you make this ppc64le specific please?  *(just add  `#[ppc64le]` to the end of the line there)

Reply:
DONE!